config/icons/.utils.ts

```diff
@@ -449,3 +450,18 @@ const flagIconList = [
 
 export const getFlagIconList = (): CountryCode[] =>
   flagIconList.map(name => name as CountryCode);
+
+const creditCardIconList = [
+  'american-express',
+  'dinners-club',
+  'discover',
+  'elo',
+  'jcb',
+  'maestro',
+  'mastercard',
+  'unionpay',
+  'visa',
+] as const;
+
+export const getCreditCardIcons = (): CreditCardTypeCardBrandId[] =>
+  creditCardIconList.map(name => name as CreditCardTypeCardBrandId);
```

Comment:
C'est peut-être naïf, mais :
```suggestion
const creditCardIconList: CreditCardTypeCardBrandId[] = [
  'american-express',
  'dinners-club',
  'discover',
  'elo',
  'jcb',
  'maestro',
  'mastercard',
  'unionpay',
  'visa',
];

export const getCreditCardIcons = () => creditCardIconList;
```

?

Reply:
ça fonctionne ?

Reply:
Oui ça fonctionne, et mieux que ça m'a fait voir une typo sur `dinners` ! Merci pour la proposition ;)